schema_based_on_diagram.sql

```diff
@@ -27,19 +27,26 @@ CREATE TABLE treatments (
 );
 
 CREATE TABLE invoices_items (
- id SERIAL PRIMARY KEY,  
- unit_price DECIMAL,
- quantity INT,
- total_price DECIMAL,
- invoices_id INT,
- treatment_id INT,
+  id SERIAL PRIMARY KEY,
+  unit_price DECIMAL,
+  quantity INT,
+  total_price DECIMAL,
+  invoices_id INT,
+  treatment_id INT,
   FOREIGN KEY (invoices_id) REFERENCES invoices(id),
   FOREIGN KEY (treatment_id) REFERENCES treatments(id)
 );
 
 CREATE TABLE medical_history_treatments(
-    treatment_id  INT,
-    medical_history_id INT,
-    FOREIGN KEY(treatment_id) REFERENCES treatments(id)
-    FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id)
-);
\ No newline at end of file
+  treatment_id  INT,
+  medical_history_id INT,
+  FOREIGN KEY(treatment_id) REFERENCES treatments(id)
```

Comment:
- [ ] Unable to create `medical_history_treatments` table due to syntax error. Kindly add a comma at the end of _line 42_.

Reply:
Hey @rloterh  while appreciating your effort in helping us to improve and make our project better, I don't think you copied the code right. because our table does have a closing parenthesis and comma. But that's okay I will delete it and add it again. Just to let you know so that you can pay a little bit more attention to your next review.

Reply:
Hello @Brenda309 

Sorry for the late response, and thanks for approaching me on this, however, I'll urge you to kindly look carefully at the inline comment above during review. It's very clear that there's no comma at the end of _line 43_. I think might be due to the fact that you online version of the code is not in sync with your code in your code editor or local machine. I'll suggest you kindly update your online version of the code.

**No comma at the end of line 43:**
![image](https://user-images.githubusercontent.com/12745474/186639645-1bcbbbcc-da20-4268-9115-6d7671b5cfc2.png)